publisher-sdk/proguard-rules.pro

```diff
@@ -42,38 +42,6 @@
 
 # Leaving Criteo SDK models.
```

Comment:
As you're working on the proguard-rules, would you mind cleaning it a little bit ? There is a lot of useless rules here.

Reply:
The retrofit rules? Sure.